syntax = "proto3";

package messenger;

//import "google/protobuf/timestamp.proto";

// common
message UserCredentials {
  string login = 1;
  string password = 2;
}

// unused TODO: implement me
message PersonalListToCreate {
  string list_name = 1;
  int64 owner_id = 2;
}

message PeopleInlistToCreate {
  int64 list_id = 1;
  int64 friend_id = 2;
}
